package.json

```diff
@@ -129,7 +129,6 @@
     "archiver": "^3.0.0",
     "browserify": "^17.0.0",
     "browserify-banner": "^1.0.15",
-    "bundle-phobia-cli": "^0.14.6",
```

Comment:
shouldn't there be a `yarn.lock` change for this? And if there is, why doesn't `git add -A && git diff --cached --exit-code` catch it?

Reply:
ohhh, `yarn install --frozen-lockfile` leaves `yarn.lock` unchanged. lol

Reply:
oops. let's remove that

Reply:
Interesting, `--fronzen-lockfile` seems to only have an error exit code when it needs to install new packages, not uninstall. It seems like any change to the `yarn.lock` file should trigger that, but I guess at least it's just unused modules sitting there, not missing ones?

Reply:
https://github.com/yarnpkg/yarn/issues/5840#issuecomment-415209523

> The first issue is the situation described by you, where `--frozen-lockfile` succeeds even if there are optimizations to be made (i.e. packages to remove). This behavior is intentional, though there are good arguments to be made that this behavior should be changed. The documentation should definitely be improved here (https://github.com/yarnpkg/yarn/issues/5847), but I don't think it's a bug.

much later...

> Just for clarity - was this ever fixed for Classic Yarn (1.x)? Facing this issue in 1.22.5 and I'd rather not have to migrate to v2.

> 1.x is frozen and will only receive security fixes, so the answer is no.

😐